### ISSUE TITLE
Upgrade to Logback 1.3.x

### ISSUE THREAD

**Issue by nehaev** [loki4j/loki-logback-appender#196]

**Comment by 3XC1T3D** [comment]
When will the pull request be released? I am getting following CVE Warning due to old logback Libs:

```
Java (jar)
==========
Total: 1 (HIGH: 1, CRITICAL: 0)

┌──────────────────────────────────────────────┬───────────────┬──────────┬────────┬───────────────────┬────────────────────────┬────────────────────────────────────────────────────────────┐
│                   Library                    │ Vulnerability │ Severity │ Status │ Installed Version │     Fixed Version      │                           Title                            │
├──────────────────────────────────────────────┼───────────────┼──────────┼────────┼───────────────────┼────────────────────────┼────────────────────────────────────────────────────────────┤
│ ch.qos.logback:logback-core                  │ CVE-2023-6481 │ HIGH     │ fixed  │ 1.2.12            │ 1.4.14, 1.3.14, 1.2.13 │ logback: A serialization vulnerability in logback receiver │
│ (binary-service-8.175.0-RC-SNAPSHOT-fat.jar) │               │          │        │                   │                        │ https://avd.aquasec.com/nvd/cve-2023-6481                  │
└──────────────────────────────────────────────┴───────────────┴──────────┴────────┴───────────────────┴────────────────────────┴────────────────────────────────────────────────────────────┘
```

**Comment by nehaev** [comment]
Hi @3XC1T3D, thank you for reporting this! I'll try to release v1.5.0-m1 with this change as soon as possible.

**Comment by nehaev** [comment]
Published current `main` as `v1.5.0-m1`.
